src/scicat_kafka.py

```diff
@@ -57,7 +57,7 @@ def build_consumer(kafka_options: KafkaOptions, logger: logging.Logger) -> Consu
     kafka_topics = collect_kafka_topics(kafka_options)
     logger.info("Subscribing to the following Kafka topics: %s", kafka_topics)
     consumer.subscribe(kafka_topics)
-    return Consumer(consumer_options)
+    return consumer
```

Comment:
Was it fast enough for ingestor? `subscribe` makes the consumer slower since `broker` tries to find the best partitian every time it polls a message.

I'm just checking since we had this problem in live data reduction: https://docs.docs.esss.dk/dmsc/groups/dram/data-reduction/data-streaming.html#assign-topicpartition-to-a-consumer-manually-and-do-not-subscribe

Reply:
> With the previous code, we were subscribing to the topics before instantiating a new consumer. 

Ah yeah I found out : D.... that was a bug I guess.. (sorry)
but I was talking about a different thing. You can either subscribe a topic or assign a topic-partition to the consumer and latter is much faster.
But since the file ingerstor is not expecting too many messages I think it'll be fine.
Just let me know if you find it too slow. We have a solution.